From 1d4ccfe0a32bba314701ea48d07308237798a764 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Sun, 13 Jul 2025 21:59:21 +0800
Subject: [PATCH 62/67] AOSCOS: net/phytium: Depends on ACPI and ARCH_PHYTIUM

The driver seems relying on ACPI to power up the device.  And the device
is only available as a part of Phytium SoC.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/phytium/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/phytium/Kconfig b/drivers/net/ethernet/phytium/Kconfig
index 14a77adbfdc1..34cbb2a0ce82 100644
--- a/drivers/net/ethernet/phytium/Kconfig
+++ b/drivers/net/ethernet/phytium/Kconfig
@@ -19,6 +19,8 @@ if NET_VENDOR_PHYTIUM

 config PHYTMAC
 	tristate "Phytium GMAC support"
+	depends on ACPI
+	depends on ARCH_PHYTIUM
 	depends on HAS_DMA
 	select PHYLINK
 	select CRC32
--
2.50.1
